Service/src/app.py

```diff
@@ -63,11 +56,19 @@ def not_found(error):
    return Response('Not Found', status=404)
 
 
-def getPortNum(defaultPort=5005):
+def argParser():
    parser = argparse.ArgumentParser()
    parser.add_argument('-p','--port', type=int, help='specify the port number')
+   parser.add_argument('-e', '--environment', type=str, help='specify the environment')
    args = parser.parse_args()
-   return args.port or defaultPort
+   return args
+
+def getPortNum():
+   return argParser().port or DEFAULT_PORT_NUM
```

Comment:
Yaaa I know, this was a conscious decision because I though the code looked cleaner this way so I don't have to make a function call outside of the main method.